src/Buffer.java

```diff
@@ -42,14 +50,18 @@ synchronized void produce(String product) {
             }
         }
         this.buffer.add(product);
+        gui.anadirTareaPorHacer(ID, product);

```

Comment:
```suggestion
        gui.anadirTareaPorHacer(id, product);
```
Cambiar a id para mantener la consistencia con la naming convention.